modules/src/main/java/org/archive/modules/fetcher/BdbCookieStore.java

```diff
@@ -143,6 +144,22 @@ public void addCookieImpl(Cookie cookie) {
             cookies.remove(key);
         }
     }
+    
+    public boolean expireCookie(Cookie cookie, Date date) {
+        byte[] key;
+        try {
+            key = sortableKey(cookie).getBytes("UTF-8");
+        } catch (UnsupportedEncodingException e) {
+            throw new RuntimeException(e); // impossible
+        }
```

Comment:
I think if we use java.nio.charset.StandardCharsets.UTF_8 we don't need to try/catch:

```suggestion
        byte[] key = sortableKey(cookie).getBytes(StandardCharsets.UTF_8);
```